cloud_functions/token_refresh/tests/test_main.py

```
@@ -0,0 +1,238 @@
+        "stream=strava": {
+            "refreshToken": mock_encrypted_token
+        }
+    }
+
+    mock_ref = MagicMock()
+    mock_ref.get.return_value = mock_doc
+
+    mock_firestore_client.collection.return_value.document.return_value = mock_ref
+
+    # Mock get_secret to return different values based on the secret name
+    def get_secret_side_effect(secret_name):
```

Ruff (0.8.2)

Missing return type annotation for private function `get_secret_side_effect`

(ANN202)